tests/test_io.py

```diff
@@ -348,6 +343,28 @@ class TestRst(TestIO):
             =====  =  =  =""").strip('\n')
     }
 
+    if (LooseVersion(tabulate.__version__) <= LooseVersion('0.7.5') or
+            platform.python_implementation() == 'PyPy'):
```

Comment:
this pypy specific difference is really weird ... need to raise that bug upstream